docs/source/markdown/podman-update.1.md.in

```diff
@@ -92,6 +92,8 @@ Changing this setting resets the timer, depending on the state of the container.
 
 @@option restart
 
+@@option ulimit
```

Comment:
This needs a extra note that the limit only takes effect on the next container restart.